doc/source/user_guide/groupby.rst

```diff
@@ -143,7 +143,7 @@ the columns except the one we specify:
 .. ipython:: python
 
    df2 = df.set_index(["A", "B"])
-   grouped = df2.groupby(level=df2.index.names.difference(["B"]))
+   grouped = df2.groupby(level="A")
```

Comment:
The line leading into this is:

> If we also have a MultiIndex on columns ``A`` and ``B``, we can group by all
the columns except the one we specify:

If we want to preserve this, one can do `[e for e in df2.index.names if e not in {"A"}]`. But I'd also be okay to remove the entire example.

Reply:
I'm worried taking union and difference here might be a common use case that we're breaking. I'm okay with removing the feature, but we could deprecate uses of these methods to make the breaking change a bit more soft. Of course, with our deprecation policy that would mean punting this off for some time.

Reply:
Fair point. If you feel strongly about it we can go through a deprecation, but the fact that FrozenList was never really publically documented (except indirectly here with this example) makes me OK with breaking this use case.

As a data point, in cudf uses `FrozenList` in `cudf.MultiIndex` to match return types but has no usage of `FrozenList.union/difference`

Reply:
I do not feel strongly here. Okay to go forward in my opinion.